controllers/quay/quayregistry_controller.go

```diff
@@ -1055,30 +1079,38 @@ func (r *QuayRegistryReconciler) updateWithCondition(
 
 func (r *QuayRegistryReconciler) cleanupPreviousSecret(log logr.Logger, ctx context.Context, quay *v1.QuayRegistry) error {
 	log.Info("deleting old objects")
-	previousSecret := &corev1.Secret{}
-	if err := r.Client.Get(
-		ctx,
-		types.NamespacedName{
-			Name:      quay.Spec.ConfigBundleSecret,
-			Namespace: quay.GetNamespace(),
-		},
-		previousSecret,
-	); err != nil {
-		r.Log.Info(fmt.Sprintf("%s secret not found, skipping", quay.Spec.ConfigBundleSecret))
-	}
-	if err := r.Client.Delete(ctx, previousSecret); err != nil {
+
+	log.Info("getting current config bundle secret")
```

Comment:
is this log correct? it seems like it's just collecting and deleting all the existing config secrets

Reply:
not sure if this log adds anything useful here, since we're not actually getting the config bundle secret, just deleting the other's. this would be more helpful if it said something along the lines of `garbage collecting outdated config secrets`

Reply:
nice catch, this is an incorrect log statement. let me update this